src/components/Countdown/Countdown.js

```diff
@@ -1,9 +1,18 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 import Timer from '../Timer/Timer'
+import { warnOnce } from '../../utils'
 
 class Countdown extends React.Component {
+  deprecationWarning() {
+    warnOnce(
+      'Countdown',
+      '"Countdown" and its variants have been deprecated. Please use "Timer" instead.'
```

Comment:
I might be wrong but I don’t think we have any variant?

```suggestion
      '"Countdown" have been deprecated. Please use "Timer" instead.'
```

Reply:
Yep, we actually don't.